README.md

```diff
@@ -127,9 +160,8 @@ Describe what a schema is, and how we represent a one-to-many relationship in a
 SQL database. If you need an example, you can use `people` and `wishes` models.
 
 Your answer...
-
 ```
-Replace this with your answer
+Schema is a blueprint for a database. one-to-many relationship is defined as a relationship between two tables that one table can have multiple matching tables.
```

Comment:
👍 
We also wanted you to mention the foreign key. For example the `Wishes` table would have a `people_id` to link back to the `id` in the person table, creating the ERD relation.